cli_test.go

```diff
@@ -58,8 +58,10 @@ type argT struct {
 	Float32 float32 `cli:"f32" usage:"type float32"`
 	Float64 float64 `cli:"f64" usage:"type float64"`
 
-	PtrBool *bool `cli:"ptrBool"`
-	PtrStr *string `cli:"ptrStr"`
+	PtrBool *bool   `cli:"ptrBool"`
+	PtrStr  *string `cli:"ptrStr"`
+
+	RequiredNotEnforced *string `cli:"*requiredNotEnforced" ignoreRequired:"true"`
```

Comment:
where is it tested?

Reply:
in all the subtests, they would have needed `--requiredNotEnforced` otherwise

Reply:
Right. Can you add subtest with this flag?

Reply:
sure